ui/app/templates/components/identity/edit-form.hbs

```diff
@@ -24,7 +24,34 @@
       />
     {{/if}}
     {{#each this.model.fields as |attr|}}
-      <FormField data-test-field={{true}} @attr={{attr}} @model={{@model}} />
+      <FormField data-test-field={{true}} @attr={{attr}} @model={{@model}}>
+        <div class="form-section">
+          {{#if this.model.canCreatePolicies}}
+            <SearchSelectWithModal
```

Comment:
This component was created to satisfy designs for the OIDC work in 1.12 and incorporating this logic into the classic SearchSelect component (since this was prior to its glimmerization) was going to take too much time and increase the complexity of what felt like an already bloated component. Also, originally the plan was for SS+Modal to only accepting a single model as an arg so that the `createRecord` was more seamlessly handled just by the component. However, this had to change because the designs included accounting for multiple policy types.

During this PR I revisited the idea of using the same component and didn't for a few reasons. However, I might have SS tunnel-vision so open to outside opinions! 

- `SearchSelect` has a couple advanced options: passing in `options` or a `search` function manually. Both felt superfluous logic to incorporate in the SS+Modal and can easily introduce bugs if handled incorrectly. (As I discovered in `path-filter-config-list.js when glimmerizing) 
- Wanted to preserve `SearchSelect` create functionality for transformation roles 
- SS handles wildcard selection

Reply:
Great thanks for the summary! I definitely agree with your points regarding the original `SearchSelect` component. I think this is something we should revisit in the future and come up with a version 2 of the component that is more simplified and flexible where we can incorporate the modal as well.